galata/test/jupyterlab/low-vision.test.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { expect, galata, test } from '@jupyterlab/galata';
+
+test.use({
+  autoGoto: false,
+  mockState: galata.DEFAULT_DOCUMENTATION_STATE,
+  viewport: { height: 1024, width: 1280 }
+});
+
```

Comment:
Is this needed?

```suggestion
test.use({
  autoGoto: false,
  mockState: galata.DEFAULT_DOCUMENTATION_STATE,
  viewport: { height: 1024, width: 1280 }
});
```

Reply:
Had a look if it was needed and decided to take it out as requested.